build/configs/artik053/avs_test/defconfig

```diff
@@ -222,7 +222,7 @@ CONFIG_ARCH_BOARD_ARTIK053=y
 # CONFIG_ARCH_BOARD_ARTIK053S is not set
 # CONFIG_ARCH_BOARD_ARTIK055S is not set
 # CONFIG_ARCH_BOARD_SIDK_S5JT200 is not set
-CONFIG_ARCH_BOARD_ARTIK05X_FAMILY=y
+CONFIG_ARCH_BOARD_FAMILY=y
```

Comment:
I think this config needs to be re-named because "architecture board family" doesn't have any meaning.

Reply:
Thanks. i fixed.